source/tutorials/bulk-writes.txt

```diff
@@ -0,0 +1,58 @@
+.. _scala-bulk-writes:
+
+=====================
+Bulk Write Operations
+=====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, multiple changes
+
+Starting in v2.6, MongoDB supports bulk write commands
+for insert, update, and delete operations in a way that allows the
+driver to implement the correct semantics for ``BulkWriteResult`` and
+``BulkWriteException``.
```

Comment:
Suggestion:
I think this content can be omitted since it references a change made to an unsupported version of MongoDB (prior to 3.4)